### PR TITLE
hope it will work

### PR DESCRIPTION
**Pull request by lolcraftboy**

### CODE REVIEW

**Comment by robske110** [comment]
I'll merge this when i have to time to modify this

**Comment by lolcraftboy** [comment]
@robske110 Thnx

**Comment by inxomnyaa** [comment]
We said we wont do this.

**Comment by lolcraftboy** [comment]
@thebigsmileXD  why y have plugin that are not fore api 2.0.0 (not all my plugins)

**Comment by robske110** [comment]
@thebigsmileXD I will implement this in a way, that noone will understand how to enable this without reading IF YOU TURN THIS ON, YOUR WORLD MAY GET CORRUPTED. ALSO IF YOU ENCOUNTER ANY BUG, PLEASE TURN THIS OPTION OFF AND CHECK IF IT STILL EXISTS (for later copy paste, yey)

**Comment by inxomnyaa** [comment]
..

**Comment by robske110** [comment]
and the reply is ...

**Comment by Aericio** [comment]
My reply would be the same.

**Comment by robske110** [comment]
...

File: src/pocketmine/resources/pocketmine.yml [hunk 1]
```diff
@@ -22,6 +22,18 @@ settings:
...
+ #Enabling these options may result in the server loading plugins that are incompatible, hence crashing the server or causing other issues
+ incompatible-plugins:
+  plugin-too-new:
+   #Load plugins whose major API version is too new
+   major: true
+   #Load plugins whose minor API version is too new
+   minor: true
+  #Load plugins whose major API version is too old
+  plugin-too-old: true
+ config-update: true
```
Comment (DaPigGuy) [inline src/pocketmine/resources/pocketmine.yml:35]: Erm 2 of em?

Comment (Aericio) [inline src/pocketmine/resources/pocketmine.yml]: Top one says Major bottom says Minor. Still don't understand it though...

Comment (DaPigGuy) [inline src/pocketmine/resources/pocketmine.yml:35]: Was talking about config-update

Comment (MCPEPIG) [inline src/pocketmine/resources/pocketmine.yml]: Was talking about config-update

**Comment by Aericio** [comment]
rip ?

**Pull request closed without merging**